packages/driver/cypress/e2e/commands/querying/querying.cy.js

```diff
@@ -776,8 +776,6 @@ describe('src/cy/commands/querying', () => {
         cy.get('#missing-el').should('have.prop', 'foo')
       })
 
-      it('throws when using an alias that does not exist')
```

Comment:
Should there be a test case added for this instead of deleted this placeholder test?

Reply:
Good thought. Was writing an assertion to cover this when it occurred to me to check on the error being thrown - dug through and found that this case is already covered here: https://github.com/cypress-io/cypress/blob/issue-7306-addQuery-custom-commands/packages/driver/cypress/e2e/commands/aliasing.cy.js#L485

It's a bit confusing that alias tests are split between `aliasing.cy.js` and `querying.cy.js`, but I don't think I want to move them as part of this PR. Another bit of cleanup to be done later.